release_images_inference.yml

```diff
@@ -16,14 +16,13 @@ release_images:
       force_release: False
   2:
     framework: "djl"
-    version: "0.23.0"
+    version: "0.24.0"
     arch_type: "x86"
     inference:
-      device_types: [ "gpu" ]
+      device_types: [ "neuronx" ]
       python_versions: [ "py39" ]
```

Comment:
Update version to ["py38"] - https://github.com/tosterberg/djl-serving/blob/master/serving/docker/pytorch-inf2.Dockerfile#L15